cs-offerings/kube-configs/chaincode_install.yaml.base

```diff
@@ -13,7 +13,7 @@ spec:
   - name: chaincodeinstall
     image: ibmblockchain/fabric-tools:1.1.0
     imagePullPolicy: Always
-    command: ["sh", "-c", "git clone -b v1.1.0 https://github.com/hyperledger/fabric $GOPATH/src/github.com/hyperledger/fabric/ && peer chaincode install -n ${CHAINCODE_NAME} -v ${CHAINCODE_VERSION} -p github.com/hyperledger/fabric/examples/chaincode/go/chaincode_example02/"]
+    command: ["sh", "-c", "git clone -b release-1.1 https://github.com/hyperledger/fabric-samples $GOPATH/src/github.com/hyperledger/fabric-samples/ && peer chaincode install -n ${CHAINCODE_NAME} -v ${CHAINCODE_VERSION} -p github.com/hyperledger/fabric-samples/chaincode/sacc"]
```

Comment:
Can you clone `v1.1.1` and not `release-1.1` so that we point to releases only

Reply:
Also I am curios why you moved to `sacc` and not change the example02 location to `https://github.com/hyperledger/fabric/tree/v1.1.1/examples/chaincode/chaintool/example02/src/chaincode`

Reply:
I have updated the config and scripts based on the review comments but instead of https://github.com/hyperledger/fabric/tree/v1.1.1/examples/chaincode/chaintool/example02/src/chaincode I have used github.com/hyperledger/fabric/examples/chaincode/go/chaincode_example02. Earlier I wasn't able to identify why chaincode_example02 was failing to instantiate (the install would always work) but figured out now that the chaincode name had to be updated from example02 to chaincode_example02 to get this working. Please pull my latest updates for review